_docs/releasing/incubation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Incubation to Graduation
+author: OST
+date: 2018-06-07
+banner:
+  image: coding.jpg
+category: Releasing
+---
+
+#### This section describes all the minimum requirements that all new projects must meet before they will be fully accepted into the main Zalando Github organisation.
+
+### Summary
+
+- Quality indicators for incubator projects
+- Review of activity
+- Review assessment
+- Graduation
+
+---
+
+### Quality indicators for incubator projects
+
+Before your start thinking about graduation, you need to make sure you are ready and meet the following requirements.
+
+- Have a clear vision and a long term plan for your project
+- Commit to be part of the Zalando Github organisation
+- Project is used and benefits to Zalando
+- Have multiple maintainers from diverse background and URM (under represented minority)
+- Project is compliant and has all the necessary documentation
+
+All these acts have an influence on your project getting graduated.
+
+### Review of activity
+
+Each project in the incubator shall undergo a regular review of progress by the Review Group. Such reviews shall occur at least quarterly. The Review Group may choose to review individual projects with greater frequency. At least one week prior to each review, the Mentor must produce a report for the Review Group detailing overall progress with a focus on the preceding review period.
+
+The Review Group will be interested in the following points for the status check. The project -
+
+- Has regular activity
+- Seeks and responds to feedback and suggestions
+- Accepts contributions
+- Responds to Issues, Pull Requests etc.
+
+### Review assessment
+
+After each review, the Review Group shall produce an assessment of the project. The assessment shall contain one of three recommendations:
+
+- that the project be Terminated;
+- that the project continues in Incubation;
+- that the project be Graduated from Incubation.
+
+### Graduation
+
+Graduation is the process where your project gets accepted into the main Zalando Github organisation. This is a democratic process, in the end, it comes down to a Vote. The road to graduation is fairly simple but does take time and effort.
+
+Prior to graduation, a project must meet the following requirements / quality indicators:
+
+- High activity: The project is highly visible and has good statistics - stars, forks etc., actively used, has regular releases or is well maintained.
+
+- Community driven: The project works to attract external contributors, builds a diverse community and is a source to get new hires for Zalando.
+
+- Promotion: The project has a website, social media channels.
+
+- Never leaves the org
```

Comment:
Clearify that zalando and projects must be aware that a graduated project never leaves the org, so there is a responsibility to maintain it long term